docs/integrator-guide.adoc

```diff
@@ -1577,6 +1626,19 @@ org.asciidoctor.integrationguide.extension.TerminalCommandExtension
 And that's all.
 Now when a `.jar` file containing the previous structure is dropped inside classpath of AsciidoctorJ, the `register` method will be executed automatically and the extensions will be registered.
 
+// From an issue somewhere but I cant find it
+
+[NOTE]
+--
+If you have installed AsciidoctorJ as recommended, asciidoctorj will be on the path, and you can use:
+
+ asciidoctorj -cp=lib/myextension.jar test.adoc
+
+If you have downloaded the distribution jars only, use a command like:
+
+ java -cp lib/asciidoctor-api-2.0.0.jar;lib/asciidoctor-core-2.0.0.jar;lib/jcommander-1.35.jar;lib/myextension.jar  org.asciidoctor.jruby.cli.AsciidoctorInvoker  test.adoc
```

Comment:
It looks like the 2 documents are not in sync, the "source" contains the jruby.jar on the classpath, this doesn't. 
Can you rerun the build please?